README.md

```diff
@@ -114,7 +118,7 @@ export AWS_SECRET_ACCESS_KEY=bar
 ```
 
 ### Add the SSH Key to the SSH Agent
-You will need to add the private key to your SSH agent: 
+Create your EC2 Key Pair with name "default" in the AWS Region you picked.  You will need to add the private key to your SSH agent: 
```

Comment:
So you find out about the "default" later in the instructions.   If you picked a different name and then you read the rest of the instructions and tried to use the default run.py, you'd have to go back to this step.   I'm trying to get the happy path and defaults to work.